### DISCUSSION TITLE
Campo Obbligatorio nel modello identità

### DISCUSSION THREAD

**Discussion by Ilmarchese1966** [globaleaks/globaleaks-whistleblowing-software#3842] [General]
Una collega chiede come mai sia stata inserita nel modello "identità" la voce obbligatoria "Metodo di contatto Alternativo" che potrebbe generare confusione a chi segnala, che potrebbe pensare che l'istruttore possa usare metodi alternativi alla piattaforma anche per avere semplici chiarimenti

**Comment by evilaliv3** [comment]
Salve, non è reso obbligatorio fornire un contatto alternativo ma è
obbligatorio dire se si ha un preferenza alternativa.

> Message ID: ***@***.***>
>
L'utente ha la possibilità di chiedere di essere contattato solo tramite
piattaforma.

Questo è utile per finalità di comprensione del fenomeno di whistleblowing
per comprendere se gli utenti possono (come crediamo) facilmente usare il
sistema o se lo hanno trovato complesso

**Comment by cienne1** [comment]
Buongiorno. Concordo con Ilmarchese1966. Il sistema consente di indicare una preferenza di contatto, ma:

1) il gestore non sa quale sia il metodo di contatto scelto dal segnalante finchè non accede all'identità
2) il segnalante potrebbe erroneamente ritenere che le comunicazioni avverranno a mezzo email e quindi non accedere più al sistema

Forse il campo relativo al metodo di contatto alternativo dovrebbe già essere visibile al gestore insieme alla segnalazione, senza legarlo all'identità.

**Reply by evilaliv3** [reply]
@cienne1: La tua soluzione ha senso in generale ma non è di fatto applicabile in quanto il numero di telefono e la mail portano facilmente all'identificazione del segnalante
